src/components/Navbar/Navbar.js

```diff
@@ -51,7 +51,8 @@ const pages = [
   [ 'Submit Project', '/nonprofits/apply' ],
   [ 'Projects', '/nonprofits' ],      
   [ 'Join Slack', '/signup'],
-  [ 'GitHub', 'https://github.com/opportunity-hack/' ]
+  [ 'GitHub', 'https://github.com/opportunity-hack/' ],
+  ['Dashboard', '/myprofile'],
```

Comment:
@Ashmitha-art - this is a great idea! how do you see this replacing the current profile or do you think this would be standalone?